.gitlab-ci.yml

```diff
@@ -0,0 +1,93 @@
+stages:
+    - build
+    - deploy
+
+.maven-build:
+    image: maven:3.5.4-jdk-8
+    stage: build
+    variables:
+        ARTIFACT_PATH: "eva-server/target/eva-$ENVIRONMENT_NAME.war"
+    script:
+        - "mvn package --settings .gitlab.settings.xml -P $MAVEN_PROFILE"
+        - "cp eva-server/target/eva.war $ARTIFACT_PATH"
+    artifacts:
```

Comment:
If we would like to add test reports, we can do it [this way](https://docs.gitlab.com/ee/ci/junit_test_reports.html#maven).